### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by fanoooo**

### CODE REVIEW

**Review by irufano — CHANGES_REQUESTED** [review]
change please

**Review by fanoooo — COMMENTED** [review]
ok

File: README.md [hunk 1]
```diff
@@ -4,13 +4,3 @@ Flutter base project by irufano.
 
 ## Getting Started
```
Comment (irufano) [inline README.md:6]: Change

**Review by fanoooo — COMMENTED** [review]
coool

**Review by irufano — APPROVED** [review]

**Review by irufano — CHANGES_REQUESTED** [review]
change

**Review by irufano — APPROVED** [review]

**Comment by irufano** [comment]
failed

**Pull request closed without merging**